tests/ops/qutrit/test_qutrit_parametric_ops.py

```diff
@@ -298,6 +354,13 @@ def test_control_wires(op, control_wires):
     assert op.control_wires == control_wires
 
 
+@pytest.mark.parametrize(
+    "op, obs, grad_fn",
+    [
+        (qml.TRX, qml.GellMann(0, 3), lambda phi: -np.sin(phi)),
+        (qml.TRY, qml.GellMann(0, 1), lambda phi: np.cos(phi)),
```

Comment:
Unrelated to this PR, but I wonder if we should consider making one of the arguments to `GellMann` a mandatory keyword argument. Every time I use it, I forget which is the index, and which is the wire!

Reply:
I would like that as well, but I have two grievances:
* Making `index` a positional argument implies that it is trainable when it isn't supposed to be.
* Keeping it as a keyword argument but placing it before `wires` also breaks convention as keyword arguments are supposed to be placed *after* `wires`.